core/api/core.api

```diff
@@ -4742,6 +4742,10 @@ public final class org/jetbrains/kotlinx/dataframe/columns/ColumnPath : java/uti
 	public fun add (Ljava/lang/String;)Z
 	public fun addAll (ILjava/util/Collection;)Z
 	public fun addAll (Ljava/util/Collection;)Z
+	public synthetic fun addFirst (Ljava/lang/Object;)V
```

Comment:
Interesting, why it was added

Reply:
I'm not sure either, probably because it extends List

Reply:
Hmm.. It seems that target jdk is now Java 21 too: https://openjdk.org/jeps/431 and not 11 as we intend. Try Xjdk-release=11 in compileKotlin freeCompilerArgs

Reply:
made an issue for it https://github.com/Kotlin/binary-compatibility-validator/issues/290

Reply:
https://kotlinlang.org/docs/compiler-reference.html#jvm-target-version
JVM target only seems to affect bytecode. jdk release limits API too

Reply:
Ah I see, but if the target jvm is 11, then it's okay to compile it against a newer version, right? Having the target at 11 allows our users to set their projects to 11 and DF still works

Reply:
I believe we need to set jdk release to 11 and toolchain to 21

Reply:
I think release and target overlap. I'll set it, but I think setting the target and sourceCompatibility does the same